apps/frontend/next.config.js

```diff
@@ -38,6 +38,7 @@ const nextConfig = {
   reactStrictMode: true,
   // experimental, however will be released the default in Nextjs 12.2.0
   swcMinify: true,
+  images: { unoptimized: true },
```

Comment:
We have to set unoptimized to true to be able to export to HTML and build with surge. I don't think it should be an issue but let me know if you have a concern @sifelix

Reply:
I don't know if it can impact amplify. I will check. I think we should give atry to the [amplify preview](https://docs.aws.amazon.com/amplify/latest/userguide/pr-previews.html) feature first.

Reply:
```
For security purposes, you can enable web previews on all apps with private repositories, but not on all apps with public repositiories. If your Git repository is public, you can set up previews only for apps that don't require an IAM service role.
```